stable/newrelic-infrastructure/values.yaml

```diff
@@ -109,3 +109,6 @@ updateStrategy: RollingUpdate
 
 # Custom attributes to be passed to the New Relic agent
 customAttribues: "'{\"clusterName\":\"$(CLUSTER_NAME)\"}'"
+
+# etcdTlsSecretName: newrelic-infra-etcd-tls-secret
```

Comment:
Why this value is commented and the next one is not?

Reply:
To be consistent with the parameters of the integration itself. The TLS secret name is always required. The namespace where it lives can be omitted and then the default value of `default` is used.

Reply:
Makes sense, thank you for the explanation.